targets/TARGET_ARM_SSG/TARGET_MUSCA_S1/objects.h

```diff
@@ -0,0 +1,76 @@
+/* mbed Microcontroller Library
```

Comment:
also mcu_boot.bin (in the target folder), is part of this ?

Reply:
It is also needed, mcuboot.bin is used in `tools\targets\ARM_MUSCA_S1.py`